docs/tutorial/getting-started.md

```diff
@@ -0,0 +1,94 @@
+# Getting Started
+
+## What you'll do
+
+- Deploy the [jenkins-k8s charm](https://charmhub.io/jenkins-k8s)
+- [Deploy and integrate agents](#deploy-and-relate-agents)
+- [Get admin password](#get-admin-password)
+
+The jenkins-k8s charm helps deploy a Jenkins automation server application with ease and
+also helps operate the charm. This
+tutorial will walk you through each step of deployment to get a basic Jenkins deployment.
+
+### Prerequisites
+
+To deploy jenkins-k8s charm, you will need a juju bootstrapped with any kubernetes controller.
+To see how to bootstrap your juju installation with microk8s, please refer to the documentation
+on microk8s [installation](https://juju.is/docs/olm/microk8s).
+
+### Setting up the tutorial model
+
+To easily clean up the resources and to separate your workload from the contents of this tutorial,
+it is recommended to set up a new model with the following command.
+
+```
+juju add-model jenkins-tutorial
+```
+
+### Deploy the jenkins-k8s charm
+
+Start off by deploying the jenkins-k8s charm. By default it will deploy the latest stable release
+of the jenkins-k8s charm.
+
+```
+juju deploy jenkins-k8s --channel=latest/edge
```

Comment:
Should the channel be here? The text above says that the default is to use the stable channel. Perhaps an explanation should be included why edge was picked in this case?

Reply:
Added a comment explanation on the channel argument, thank you!